src/ui-lib/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss

```diff
@@ -72,12 +70,11 @@ body > header {
     // Overflow, so that right edge of navlinks is flush
     // with screen edge
     width: calc(100% + #{$grav-page-content-inset});
```

Comment:
This oversized width is no longer needed since we don't have those borders anymore. This should have actually been removed a while ago - it's nothing to do with this PR. However, since we're here, could you please clean it up. Thanks!